static/js/components/About.jsx

```diff
@@ -79,40 +114,63 @@ BibLink.propTypes = {
 };
 
 const About = () => {
+  // {% block componentConsts -%}
   const classes = useStyles();
   const version = useSelector((state) => state.sysInfo.version);
   const cosmology = useSelector((state) => state.sysInfo.cosmology);
   const cosmoref = useSelector((state) => state.sysInfo.cosmoref);
   const gitlog = useSelector((state) => state.sysInfo.gitlog);
+  // {% endblock -%}
+
+  // {% block configVariables -%}
+  // Replace these with Jinja double-brace injection when extending if you just
+  // want to use a config file variable instead of this default appTitle without
+  // overwriting a render block that uses it below.
+  //
+  // This is done like this instead of just directly writing something like
+  // {{ app.title }} below because the actual double-brace injection syntax will
+  // not be valid JSX if not extending the About component.
+  const appTitle = "SkyPortal";
+  // {% endblock -%}
 
   return (
-    <Box className={classes.root}>
-      <h2>
-        This is SkyPortal&nbsp;
+    <Paper className={classes.root}>
+      {/* {% block header -%} */}
+      <Typography variant="h5">
+        This is {appTitle}&nbsp;
         <code>v{version}</code>.
-      </h2>
-      <p>
+      </Typography>
+      {/* {% endblock -%} */}
+      {/* {% block projectIntro -%} */}
+      <Typography variant="body1">
         The project homepage is at&nbsp;
         <a href="https://skyportal.io">https://skyportal.io</a>
-      </p>
-      <p>
+      </Typography>
+      {/* {% endblock -%} */}
+      {/* {% block documentation -%} */}
+      <Typography variant="body1">
         Documentation lives at&nbsp;
         <a href="https://skyportal.io">https://skyportal.io/docs/</a>
-      </p>
-      <p>
-        You may also interact with SkyPortal through its API. Generate a token
+      </Typography>
+      <Typography variant="body1">
+        You may also interact with {appTitle} through its API. Generate a token
         from your&nbsp;
         <Link to="/profile">profile</Link>&nbsp; page, then refer to the&nbsp;
         <a href="https://skyportal.io/docs/api.html">API documentation</a>.
```

Comment:
Likewise, should import doc URL from config.yaml

Reply:
Ah ok, sure I think moving these to the configVariables block above (like with the appTitle) is good. In this case, none of the URL's were actually in the configs before this PR either so I was just carrying that over. But again, I would push back on directly importing from the configs here for the reasons I stated above.

Reply:
From the point of fritz, it's sufficient to have these overridable (Fritz can rewrite with its own paragraph).

This paragraph is not really applicable to other projects, so think using SkyPortal by default makes sense here.

Reply:
Actually, with regards to getting doc URL from config.yaml, do we really want that? If we make that change, why not also get the github URL and the project homepage URL? I think it'll start bloating the configs unnecessarily. As Stefan says, these segments will likely be overridden by Fritz and other applications anyways so I don't know if moving the URLs to the configs are worth it.